data/human/deep missions.txt

```diff
@@ -2752,3 +2752,1453 @@ mission "Stone of our Fathers 5"
+					goto fixme
+			`	You have much more faith in your ship than some antique vehicle. You spend hours ripping once-vital machinery out of the Ice-Crawler in order to make a signal booster for your comms, eventually cobbling together a functional booster and programming a repeating burst S.O.S. message into it.`
+			`You're certain your ship will eventually pick it up. Unfortunately, the damage you've done to the Ice-Crawler has affected its heater, which gives out during your eighth hour of waiting. In the meantime, you put on the jacket Hjlod left you and stuff it with rags and trash in a bid to keep warm.`
+			choice
+				`	(Try to ignore the cold and get some sleep.)`
+			`	It has been 19 hours since the avalanche, and you've lost feeling in your hands and feet. There are no rations, and what stale warmth was once here is now gone. Hopefully, you didn't make a mistake waiting here. You know you made a mistake by going on this miserable tour. You just want to be warm again, and you certainly don't want to die here.`
+			`	You begin to fade in and out of consciousness, when suddenly you feel a deep vibration, and hear a low rumble from outside.`
+			`	There is no memory of what happens next. But somehow you must have crawled out of that ice buried Ice-Crawler, through the snow, and into the hatch of your ship. Because the next thing you do remember is laying in your ship, warming up, and being glad that the ordeal is over.`
+				decline
+			label fixme
+			`	After hours of tinkering with the guts of the machine, you've managed to repair most of it, and all it cost you was one bad burn on your arm. Now, what it comes down to is a sequence of wires. If you can plug these in correctly, the Ice-Crawler should start back up.`
+			choice
+				`	(White, green, blue.)`
+					goto boom
+				`	(Green, white, blue.)`
+					goto greatsuccess
+				`	(Blue, green, white.)`
+					goto boom
+			label boom
+			` 	You're confident that this is the correct sequence of wires. You plug each one in and start the Ice-Crawler up. The vehicle hums to life.`
+			choice
+				`	(Pump your fist.)`
+				`	(Cheer.)`
+					goto cheer
+			` 	In a moment of levity, you pump your fist into the air.`
+			`	Unfortunately, your fist does not make it very far. The Ice-Crawler, with its criss-crossed wires, has a critical energy build up that results in an instantaneous, completely deadly explosion.`
+			scene "scene/smeer"
+			`	The Ice-Crawler, your entire body, your fist, and more than a couple hundred tons of snow are all incinerated in a fraction of a second.`
+				die
+			label cheer
+			`	The sense of accomplishment inside you is so overwhelming that you find yourself unable to contain it. A cry of pure joy leaps out of your mouth. Or begins too. Unfortunately, it doesn't get too far. As the Ice-Crawler, with its criss-crossed wires, has a critical energy build up that results in an instantaneous, completely deadly explosion.`
+			scene "scene/smeer"
```

Comment:
Same as above.

Reply:
I have replaced this image with a new one. Does it still feel unnecessary? If you do not like the new picture I will remove the /smeer image entirely.